docs/CodeOfConduct.md

```diff
@@ -55,8 +55,10 @@ a project may be further defined and clarified by project maintainers.
 ## Enforcement
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
-reported by contacting any member of the [Community Governance Committee](
-Committers.md#Committee_Members). All complaints will be reviewed and investigated
+reported by contacting the [Community Governance Committee](Committers.md)
+at oesdkcgc@lists.confidentialcomputing.io, or by contacting any individual
```

Comment:
This doesn't actually work (similarly for the recommendation in docs/Committers.md), the list automatically rejects non-member mails as a tradeoff for not moderating mails to the list (which the oesdk list does) We should consider setting up a separate oesdkcoc mailing list or alias?

Reply:
Is there a reason people should not be able to directly send mail to the oesdkgcg list? Especially if oesdkcoc and oesdkgcg will go to the same people.

Reply:
My opinion: I think anyone should be able to send email to oesdkcgc.   Since all the CGC members currently have a public email address listed on the https://github.com/openenclave/openenclave/blob/master/docs/Committers.md page, I don't think there's significant benefit in preventing it or moderating it, given that anyone could just send email to the set of individual email addresses directly.